README.md

```diff
@@ -100,12 +99,10 @@ To configure the collection for your Tableau environment:
    10ay.online.tableau.com
    ```
 
-2. Select the authentication method you wish to use and declare the `CURRENT VALUE` for those fields in the environment file. A detailed description of available methods can be found in the [Authentication section of the API Reference](https://help.tableau.com/current/api/rest_api/en-us/REST/rest_api_ref_authentication.htm#sign_in). Authentication methods include Username & Password, [Personal Access Token](https://help.tableau.com/current/online/en-us/security_personal_access_tokens.htm) (PAT), and JWT (Connected Apps).
+2. Determine the authentication method you wish to use and declare the `CURRENT VALUE` for those fields in the environment file. You can find a list of the variables for your authentication method in [Tableau authentication credential and permissions](#tableau-authentication-credential-and-permissions). 
 
    To understand how credentials impact the responses to obtain from Tableau's REST API, see [Tableau authentication credentials and permissions](#tableau-authentication-credential-and-permissions).
```

Comment:
Suggest:

To see a list of available authentication variables and understand how users' credentials impact the content of Tableau's REST API responses, see [Tableau authentication credentials and permissions](#tableau-authentication-credential-and-permissions).